### ISSUE TITLE
ML9 version - No ability to define ML endpoint to write to

### ISSUE THREAD

**Issue by jonesyface** [sanjuthomas/kafka-connect-marklogic#15]
In the ml8 version of the connector there is a ml.connection.url property that let's you specifically define the endpoint you'd like to write to. 

However, the ml9 version instead has ml.connection.host and ml.connection.port properties, which by default uses the /v1/documents endpoint. The documentation says:
"By default the /v1/documents endpoint at port 8000 is used. You may change that in the marklogic-sink.properties file. You may use your own REST/Service extension instead of the out of the box document API to do any transformation on the way in."

But unless I'm missing something, I can't see a way of defining a REST extension to use instead of the default?

**Comment by sanjuthomas** [comment]
@jonesyface , yes. you are right. in the ML 9 version, we don't have that capability since we were using the ML java client library instead of the apache common http library. I think we can make that change.

**Comment by dixitsingla** [comment]
@sanjuthomas Does marklogic kafka connector now supports ml.connection.url property. In our case we need to get the data from kafka and then store to Marklogic through custom end point (Wanted to use the Store API we have created in our project).

Also, could you please confirm how to apply the transformation before sending the data to Marklogic. From Kafka data will be fetched in Json format and we need to convert it to structured (as per schema) XML and send to custom Store API.
